packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -24,7 +24,8 @@ import {
 } from '@opentelemetry/core';
 import {ExporterOptions} from './external-types';
 import {GoogleAuth, JWT} from 'google-auth-library';
-import {google, monitoring_v3} from 'googleapis';
+import {monitoring} from 'googleapis/build/src/apis/monitoring';
```

Comment:
nit a comment would be good

```suggestion
// Import directly from this module instead of googleapis to improve bundler tree shaking
import {monitoring} from 'googleapis/build/src/apis/monitoring';
```